mingus/extra/fft.py

```diff
@@ -40,6 +40,9 @@
 # So before we do any performance critical calculations we set up a cache of all
 # the frequencies we need to look up.
 
+def xrange(x):
+    return iter(range(x))
+
```

Comment:
Seems this is not used... If you really need it, you should instead import `range` from `six.moves`

Reply:
Was probably used in a previous version of mingus, I certainly ran into an undefined xrange function error at some point